spec/models/volunteer_spec.rb

```diff
@@ -24,4 +24,31 @@
     expect(volunteer.errors[:name][0]).to eq("can't be blank")
     expect(volunteer.errors[:about][0]).to eq("can't be blank")
   end
+
+  describe "Volunteer groups" do
+    it "has collection of groups" do
+      volunteer_groups = Volunteer.groups
+      groups = {
+        "hospital" => "help nurses in hospitals",
```

Comment:
Same comment: 
> We could use the same values as the keys. The values here will be written to the db.

Reply:
@hantuzun Then we have the issue of how we displayed them in the UI are we OK with just displaying one word descriptions int he dropdown?